database/leveldb/db.go

```diff
@@ -60,6 +63,18 @@ const (
 	// levelDBByteOverhead is the number of bytes of constant overhead that
 	// should be added to a batch size per operation.
 	levelDBByteOverhead = 8
+
+	// minCompactionL0Trigger is the minimum value for CompactionL0Trigger.
```

Comment:
Have we observed that two of these parameters are needed to be changed at the same time? 

Have we seen what happens when only one of them is changed?

Reply:
You just get less variance if you do this, but I think we're shooting for the maximum possible variance.

Reply:
I get that, but if (for example) variable A is responsible for 99% of the variance and variable B is responsible for 1% of it then it makes sense to only tune A.